curriculum/challenges/english/08-coding-interview-prep/data-structures/size-of-the-set.english.md

```diff
@@ -6,12 +6,12 @@ challengeType: 1
 
 ## Description
 <section id='description'>
-In this exercise we are going to create a size function for our Set. This function should be named <code>this.size</code> and it should return the size of the collection.
+  Set class should also have a <code>size()</code> method which returns the size of the collection.
 </section>
 
 ## Instructions
 <section id='instructions'>
-
+  In this exercise we are going to create a <code>size</code> function for our Set. This function should be named <code>this.size</code> and it should return the size of the collection.
```

Comment:
My suggestion is to write the instructions in second person and one other small change.
```suggestion
In this exercise you are going to create a <code>size</code> function for the `Set` class. This function should be named <code>this.size</code> and it should return the size of the collection.
```